Adding groups documentation (#32)
ElixirTeSS/docs: TeSS_docs/groups/intro-groups.md

# Introduction to groups

Groups is a feature which for now is used only to manage access of private spaces.

> A user being part of a group has access to every private space which requires the group.

Reviewer: ```suggestion
A user of a group has access to every private space associated with that group.
```

Each group has multiple owners. A owner can add and remove people to the group.

To request a new group on TeSS for your use case:

1. Select a TeSS instance for your request.

> 2. Prepare the following details of the new group: a title, your username and the list of users/owners of your group

Reviewer: ```suggestion
2. Prepare the following details of the new group: a title, your username and the list of users/owners of your group.
```
